website_sale_product_minimal_price/controllers/main.py

```diff
@@ -23,55 +22,25 @@ def get_combination_info_pricelist_atributes(
         today = fields.Date.today()
         res = []
         pricelist = current_website.get_current_pricelist()
-        pricelist_variant_items = pricelist.item_ids.filtered(
+        pricelist_items = pricelist.item_ids.filtered(
             lambda i: i.product_id.id == product.id
+            or i.product_tmpl_id.id == product.product_tmpl_id.id
```

Comment:
I think the order is relevant here, as having a rule for template and for variant makes one of them invalid, and you are not taking this into account here.

Reply:
I take care about that on `_prepare_dictionary` method.

Reply:
I'm still not seeing any difference on this matter.

Imagine this pricelist:

- Template with min_qty = 1 - Price 10
- Template with min_qty = 30 - Price 5
- Variant with min_qty = 1 - Price 11
- Variant with min_qty = 5  - Price 10
- Variant with min_qty = 40  - Price 6

My guess is that this is going to paint

>=1 - 11 €
>=1 - 10 €
>=5 - 10 €
>=30 - 5 €
>=40 - 6 €